Taskfile

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+if [[ -z "${HISTORIC_TAG_DATA_PATH}" ]]; then
+    echo "[WARNING] You need to define HISTORIC_TAG_DATA_PATH, that will contain historic tag data. Exiting."
+    exit 1
+fi
+
+function build {
+    docker build -t daccs-image-updater:0.1 .
+}
+
+function run {
+    if [[ -z "${GITHUB_USER}" ]]; then
+        echo "[ERROR] Missing GITHUB_USER. Exiting."
+        exit 1
+    fi
+
+    if [[ -z "${GITHUB_PASSWORD}" ]]; then
+        echo "[ERROR] Missing GITHUB_PASSWORD. Exiting."
+        exit 1
+    fi
+
+    docker run -v ${HISTORIC_TAG_DATA_PATH}:/opt/local/src/daccs-image-updater/data \
+     -v ~/.gitconfig:/etc/gitconfig \
```

Comment:
Why volume mount user `.gitconfig` to `/etc/gitconfig`?  If this is going to be eventually a cron job on some server, should it be completely standalone as much as possible, without relying on user config?

Reply:
Was for git push related stuff, removed since now using GITHUB_TOKEN